includes/api/class-wc-admin-rest-reports-customers-stats-controller.php

```diff
@@ -0,0 +1,353 @@
+<?php
+/**
+ * REST API Reports customers stats controller
+ *
+ * Handles requests to the /reports/customers/stats endpoint.
+ *
+ * @package WooCommerce Admin/API
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST API Reports customers stats controller class.
+ *
+ * @package WooCommerce/API
+ * @extends WC_REST_Reports_Controller
+ */
+class WC_Admin_REST_Reports_Customers_Stats_Controller extends WC_REST_Reports_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v4';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'reports/customers/stats';
+
+	/**
+	 * Maps query arguments from the REST request.
+	 *
+	 * @param array $request Request array.
+	 * @return array
+	 */
+	protected function prepare_reports_query( $request ) {
+		$args                        = array();
+		$args['registered_before']   = $request['registered_before'];
+		$args['registered_after']    = $request['registered_after'];
+		$args['match']               = $request['match'];
+		$args['name']                = $request['name'];
+		$args['username']            = $request['username'];
+		$args['email']               = $request['email'];
+		$args['country']             = $request['country'];
+		$args['last_active_before']  = $request['last_active_before'];
+		$args['last_active_after']   = $request['last_active_after'];
+		$args['orders_count_min']    = $request['orders_count_min'];
+		$args['orders_count_max']    = $request['orders_count_max'];
+		$args['total_spend_min']     = $request['total_spend_min'];
+		$args['total_spend_max']     = $request['total_spend_max'];
+		$args['avg_order_value_min'] = $request['avg_order_value_min'];
+		$args['avg_order_value_max'] = $request['avg_order_value_max'];
+		$args['last_order_before']   = $request['last_order_before'];
+		$args['last_order_after']    = $request['last_order_after'];
+
+		$between_params = array( 'orders_count', 'total_spend', 'avg_order_value' );
```

Comment:
`last_active` and `registered` are missing from this array. They are also missing in the `/reports/customers` endpoint. Perhaps a clue as to why thats not working

Reply:
Created an issue to track separately here https://github.com/woocommerce/wc-admin/issues/1388